src/core/cpuid.d

```diff
@@ -980,57 +1002,57 @@ shared static this()
     // Set the immortals
 
     _dataCaches =     datacache;
-    _vendor =         cast(string)vendorID;
-    _processor =      processorName;
-    _x87onChip =      (features&FPU_BIT)!=0;
-    _mmx =            (features&MMX_BIT)!=0;
-    _sse =            (features&SSE_BIT)!=0;
-    _sse2 =           (features&SSE2_BIT)!=0;
-    _sse3 =           (miscfeatures&SSE3_BIT)!=0;
-    _ssse3 =          (miscfeatures&SSSE3_BIT)!=0;
-    _sse41 =          (miscfeatures&SSE41_BIT)!=0;
-    _sse42 =          (miscfeatures&SSE42_BIT)!=0;
-    _sse4a =          (amdmiscfeatures&SSE4A_BIT)!=0;
-    _aes =            (miscfeatures&AES_BIT)!=0;
-    _hasPclmulqdq =   (miscfeatures&PCLMULQDQ_BIT)!=0;
-    _hasRdrand =      (miscfeatures&RDRAND_BIT)!=0;
+    _vendor =         cast(string)cf.vendorID;
```

Comment:
would it make sense to put immutables into a struct too? Then these lines are trivially re-written as `icf = Immortals(cast(string)cf.vendorID, ...); `